components/tools/OmeroPy/src/omero/util/populate_metadata.py

```diff
@@ -1605,10 +1631,14 @@ def _write_to_omero_batch(self, to_delete, loops=10, ms=500):
         # an exception has been thrown (likely LockTimeout)
         rsp = callback.getResponse()
         if isinstance(rsp, omero.cmd.OK):
```

Comment:
Yes we found out later that the problem is in

```
ndal = len(rsp.deletedObjects.get(
     "ome.model.annotations.AnnotationLink", []))
```

there is no `ome.model.annotations.AnnotationLink`